source/NVDAObjects/UIA/edge.py

```diff
@@ -425,7 +426,7 @@ def _get_role(self):
 		role=super(EdgeNode,self).role
 		if not isinstance(self,EdgeHTMLRoot) and role==controlTypes.ROLE_PANE and self.UIATextPattern:
 			return controlTypes.ROLE_INTERNALFRAME
-		ariaRole=self._getUIACacheablePropertyValue(UIAHandler.UIA_AriaRolePropertyId)
+		ariaRole=self._getUIACacheablePropertyValue(UIAHandler.UIA_AriaRolePropertyId).lower()

```

Comment:
This change isn't related to landmark reporting, but since edge threads mixed case roles as valid, I had to convert the aria role to lower case for landmarks. Changed this here for consistency.